### PR TITLE
Renames the gem and associated stuff

### PR DESCRIPTION
**Pull request by benlovell**
Like an absolute noddy, I forgot to check rubygems.org when I named
this project `rad_core`, and subsequently found out it was taken (by a
project that has been dead since 2011, but let's not go there)

Followed the convention of prefixing the gem `mas-`.

Lesson learned.

### CODE REVIEW

File: spec/dummy/config/application.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require 'rails/all'
 
 Bundler.require(*Rails.groups)
-require "rad_core"
+require "mas/rad_core"
```
Comment (tomas-stefano) [inline spec/dummy/config/application.rb]: double quotes here.

File: spec/dummy/config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,3 @@
 Rails.application.routes.draw do
-
-  mount RadCore::Engine => "/rad_core"
+  mount MAS::RadCore::Engine => "/rad_core"
```
Comment (tomas-stefano) [inline spec/dummy/config/routes.rb]: Double quotes here.

File: spec/dummy/config/application.rb (hunk 1 above)
Comment (benlovell) [inline spec/dummy/config/application.rb]: I wanted to pick these all up in another PR. This one just includes the exact changes to accommodate the change in naming.

Also - thanks, Rails generators for your insistence on mixed quotes.

File: spec/dummy/config/routes.rb (hunk 2 above)
Comment (benlovell) [inline spec/dummy/config/routes.rb]: Same here.

File: spec/dummy/config/application.rb (hunk 1 above)
Comment (tomas-stefano) [inline spec/dummy/config/application.rb]: Ok. different PR then :).

**Comment by tomas-stefano** [comment]
:+1:

**Comment by jongilbraith** [comment]
:+1:

**Pull request merged**